### PR TITLE
Feature/standardized billing line item model

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
## PR Overview
**This PR will address the following Issue/Feature:** Internal Project

**This PR will result in the following new package version:** `v0.14.0`

While there are no immediate breaking changes in this PR, it will include a net new end model as well as two new upstream staging models `discount` and `product` which should be accompanied by a minor (v0.X.0) release.

**Please provide the finalized CHANGELOG entry which details the relevant changes included in this PR:**

> ## Feature Updates
> - Addition of the `stripe__line_item_enhanced` model. This model builds a standard denormalized analytical table which can be used to report on your revenue, subscription, customer, and product metrics from your billing platform. This model has been designed to match the schema of the similarly named `*__line_item_enhanced` model within the Shopify, Recharge, Recurly, and Zuora platforms in an effort to standardize reporting when using data from a billing platform.

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [X] dbt run –full-refresh && dbt test
- [X] dbt run (if incremental models are present) && dbt test

Before marking this PR as "ready for review" the following have been applied:
- [n/a] The appropriate issue has been linked, tagged, and properly assigned
- [X] All necessary documentation and version upgrades have been applied
    
- [will be generated before release] docs were regenerated (unless this PR does not include any code or yml updates)
- [X] BuildKite integration tests are passing
- [X] Detailed validation steps have been provided below

### Detailed Validation
Please share any and all of your validation steps:

Validation tests have been applied for the new end model and we are opening this feature to the community to receive feedback before broader release.

### If you had to summarize this PR in an emoji, which would it be?
 
💬

### CODE REVIEW

File: packages.yml [hunk 1]
```diff
@@ -1,3 +1,7 @@
 packages:
-- package: fivetran/stripe_source
-  version: [">=0.11.0", "<0.12.0"]
\ No newline at end of file
+# - package: fivetran/stripe_source
+#   version: [">=0.12.0", "<0.13.0"]
+
+- git: https://github.com/fivetran/dbt_stripe_source.git
+  revision: feature/standardized-billing-line-item-model
+  warn-unpinned: false
```
Comment (fivetran-joemarkiewicz) [inline packages.yml:7]: Reminder to update before merging.

File: models/standardized_models/stripe__line_item_enhanced.sql [hunk 2]
```diff
@@ -0,0 +1,294 @@
+{{ config(
+    enabled=(
+        var('stripe__standardized_billing_model_enabled', False) and 
+        (var('stripe__using_invoices', True) and var('stripe__standardized_billing_model_enabled', True))
```
Comment (fivetran-catfritz) [inline models/standardized_models/stripe__line_item_enhanced.sql:4]:
```suggestion
        (var('stripe__using_invoices', True) and var('stripe__standardized_billing_model_enabled', False))
```
This should be disabled by default.

**Review by fivetran-catfritz — COMMENTED** [review]
Just one update!

**Review by fivetran-catfritz — APPROVED** [review]
lgtm!

**Review by fivetran-reneeli — APPROVED** [review]
realized i never approved this!

**Pull request merged**